### PR TITLE
PLFM-3628

### PR DESCRIPTION
**Pull request by brucehoff**

### CODE REVIEW

File: integration-test/src/test/java/org/sagebionetworks/ITCloudMailIn.java [hunk 1]
```diff
@@ -176,24 +176,6 @@ public void testCloudMailInMessageWrongCredentials() throws Exception {
 		assertEquals(HttpStatus.SC_UNAUTHORIZED, response.getStatusCode());
 	}
 
-	// send from an invalid email and check that the error comes back
```
Comment (brucehoff) [inline integration-test/src/test/java/org/sagebionetworks/ITCloudMailIn.java:179]: This repeats a test at the manager level.

File: lib/lib-doi/src/test/java/org/sagebionetworks/doi/datacite/DataciteMetadataTranslatorTest.java [hunk 2]
```diff
@@ -288,29 +288,6 @@ public void testEmptyCreatorName() {
 		validateDoiCreator(creator);
 	}
 
-	@Test
```
Comment (brucehoff) [inline lib/lib-doi/src/test/java/org/sagebionetworks/doi/datacite/DataciteMetadataTranslatorTest.java:291]: PLFM-5145 was closed as won't do, so we remove this

File: services/repository-managers/src/test/java/org/sagebionetworks/repo/queryparser/QueryParserTest.java [hunk 3]
```diff
@@ -367,15 +367,6 @@ public void testMissingTable() throws Exception {
 		queryShouldFail("select * from order by creationDate desc", null);
 	}
 
-	/**
```
Comment (brucehoff) [inline services/repository-managers/src/test/java/org/sagebionetworks/repo/queryparser/QueryParserTest.java:370]: this query system is deprecated

File: services/repository/src/test/java/org/sagebionetworks/repo/web/controller/DefaultControllerAutowiredAllTypesTest.java [hunk 4]
```diff
@@ -368,31 +368,6 @@ public void testUpdateEntity() throws Exception{
 		}
 	}
 	
-	@Ignore
-	@Test
-	public void testGetPath() throws Exception{
```
Comment (brucehoff) [inline services/repository/src/test/java/org/sagebionetworks/repo/web/controller/DefaultControllerAutowiredAllTypesTest.java:373]: We have an EntityTypeUtilsTest

File: services/repository/src/test/java/org/sagebionetworks/repo/web/controller/DefaultControllerAutowiredAllTypesTest.java [hunk 5]
```diff
@@ -476,66 +451,6 @@ public void testUpdateEntityAcl() throws Exception {
 
 	}
 	
-	@Ignore
-	@Test
```
Comment (brucehoff) [inline services/repository/src/test/java/org/sagebionetworks/repo/web/controller/DefaultControllerAutowiredAllTypesTest.java:480]: we test ACL functionality at the manager level

File: services/repository/src/test/java/org/sagebionetworks/repo/web/controller/ObjectTypeSerializerTest.java [hunk 6]
```diff
@@ -24,22 +23,20 @@
...
 import org.sagebionetworks.schema.adapter.org.json.EntityFactory;
 import org.sagebionetworks.schema.adapter.org.json.JSONObjectAdapterImpl;
-import org.springframework.beans.factory.annotation.Autowired;
 import org.springframework.http.HttpHeaders;
 import org.springframework.http.HttpOutputMessage;
 import org.springframework.http.MediaType;
 import org.springframework.http.converter.HttpMessageNotWritableException;
 
-public class ObjectTypeSerializerTest extends AbstractAutowiredControllerTestBase {
+public class ObjectTypeSerializerTest {
```
Comment (brucehoff) [inline services/repository/src/test/java/org/sagebionetworks/repo/web/controller/ObjectTypeSerializerTest.java:31]: This doesn't need to be an autowired test.

File: services/repository/src/test/java/org/sagebionetworks/repo/web/controller/ObjectTypeSerializerTest.java [hunk 7]
```diff
@@ -63,7 +60,7 @@ public void testJson(){
 		assertNotNull(clone);
 		assertEquals(project, clone);
 	}
-	@Ignore
+
```
Comment (brucehoff) [inline services/repository/src/test/java/org/sagebionetworks/repo/web/controller/ObjectTypeSerializerTest.java:63]: This test passes.

**Review by john-hill — APPROVED** [review]

**Pull request merged**